tests/test_on_data/test_gin_ecephys.py

```diff
@@ -132,19 +143,35 @@ class TestConverter(NWBConverter):
                     interface_kwargs=interface_kwargs,
                 )
             )
-    for suffix in ["ap", "lf"]:
+
+    for suffix, spikeextractors_backend in product(["ap", "lf"], [True, False]):
```

Comment:
@CodyCBakerPhD 
So, I did not notice until now but this was in the previous form of the test already.
Should we be really testing both "ap" and "lf" signals using `SpikeGLXRecordingInterface` for both?
Shouldn't we use `SpikeGLXLFPInterface` for `lf`?
Was there a reason for having it like this or is it an oversight?

Reply:
You can safely remove it, that's likely from the days before the lfp specific test class

Reply:
All right, done.